test/integration/controllermanager/shoot/quota/quota_test.go

```diff
@@ -184,8 +184,14 @@ var _ = Describe("Shoot Quota controller tests", func() {
 		metav1.SetMetaDataAnnotation(&shoot.ObjectMeta, "shoot.gardener.cloud/expiration-timestamp", time.Now().UTC().Add(48*time.Hour).Format(time.RFC3339))
 		Expect(testClient.Patch(ctx, shoot, patch)).To(Succeed())
 
-		fakeClock.Step(49 * time.Hour)
+		By("Verify that shoot is not deleted after original expiration time (1 day) has passed")
```

Comment:
This might be flaky if the manager's cache isn't fast enough to observe the above annotation update.
But this is just a guess.

Reply:
Yeah, let's get it in for now and see. I will keep an eye on it :)